Automattic/wp-calypso
# FSE - force private by default if in test-fse flow

#### Changes proposed in this Pull Request

Force the privateByDefault for users on `test-fse` flow in order to bypass 90/10 AB test split.

#### Testing instructions

* Apply the patch to your FSE test environment
* Add some debugging to https://github.com/Automattic/wp-calypso/blob/master/client/lib/signup/private-by-default.ts#L19 to check that it bypasses abtest() on `test-fse` flow and returns true - site should always be created as private. The normal abtest() path should be followed when not on test-fse flow
* to test FSE flow start with https://dev-env/start/test-fse
